code/game/gamemodes/meteor/meteor.dm

```diff
@@ -24,10 +24,10 @@
 
 
 /datum/game_mode/meteor/proc/sendmeteors()
-	var/waveduration = world.time + rand(0,1000) + text2num("[wave]000")
-	var/waitduration = rand(3000,9000)
+	var/waveduration = world.time + rand(0,1000) + text2num("[wave]000" / 2)

```

Comment:
Yeah making it 
```c
var/waveduration = world.time + rand(0,1000) + text2num("[wave]000") / 2
```
seems to make it work in a way that I assume is intended, I'll let you make the PR